### PR TITLE
Docs clarification on naming jinja filters file

### PR DESCRIPTION
**Pull request by jmcgill298**
### Fixes: #<ISSUE NUMBER GOES HERE>

Proposing a clarification in docs

### CODE REVIEW

File: nautobot/docs/plugins/development.md [hunk 1]
```diff
@@ -589,7 +589,7 @@ template_extensions = [SiteAnimalCount]
 
 Plugins can define custom Jinja2 filters to be used when rendering templates defined in computed fields. Check out the [official Jinja2 documentation](https://jinja.palletsprojects.com/en/3.0.x/api/#custom-filters) on how to create filter functions.
 
-In the file that defines your filters, you must import the `library` module from the `django_jinja` library. Filters must then be decorated with `@library.filter`. See an example below that defines a filter called `leet_speak`.
+In the file that defines your filters (specified in the [PluginConfig](#pluginconfig-attributes)), you must import the `library` module from the `django_jinja` library. Filters must then be decorated with `@library.filter`. See an example below that defines a filter called `leet_speak`.
```
Comment (glennmatthews) [inline nautobot/docs/plugins/development.md:592]: Perhaps?

```suggestion
In the file that defines your filters (by default `jinja_filters.py`, but configurable in the [PluginConfig](#pluginconfig-attributes) if desired), you must import the `library` module from the `django_jinja` library. Filters must then be decorated with `@library.filter`. See an example below that defines a filter called `leet_speak`.
```

**Review by glennmatthews — APPROVED** [review]
Thank you for the PR! One suggested change in phrasing, otherwise looks great.

**Comment by jmcgill298** [comment]
Thanks @glennmatthews I have implemented the suggestiong

**Review by jathanism — APPROVED** [review]
Thanks @jmcgill298 for the contribution!

**Pull request merged**